android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -98,6 +105,11 @@ public void init(Context context, IVideoStreamListener outputListener, VideoStre
 
         mOutputListener = outputListener;
 
+        SharedPreferences p = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
@santhanamk this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?

Reply:
@kostyaBoss I still see this.  On line 106, can you rename the variable `p` to `preferences`?